dist/async-test.js

```diff
@@ -1,3 +1,9 @@
+(function (global, factory) {
```

Comment:
Also while it is nice to see final product, do revert dist folder. before checking in.